src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -19,6 +19,17 @@ abstract class PipelineTargets(config: Config) {
       "writeOpsMetrics, lastOptimizedTS, Pipeline_SnapTS, primordialDateString").split(", ")
   )
 
+  val apiEventsTarget: PipelineTable = PipelineTable(
+    name = "api_events_meta",
+    _keys = Array("organization_id", "moduleId", "Overwatch_RunID"),
+    config = config,
+    partitionBy = Array("organization_id"),
```

Comment:
suggest partitioning by org_id and perhaps endpoint or something. Develop appropriate partitioning scheme for this output.

Reply:
done added end_point in the partitionBy.